### ISSUE TITLE
Cum being purple/pixilated and broken

### ISSUE THREAD

**Issue by ghost** [Archival1/CV-BugReporting#21]
**Describe the bug**
When churning any prey the cun is either purple or reflectively clear, could be caused by me being on a laptop unsure how to fix if possible

**To Reproduce**
1. any level
2. any target
3. unknown
4. n/a

Macbook pro
2.6 GHz 6-Core Intel Core i7
Intel UHD Graphics 630 1536 MB

[https://cdn.discordapp.com/attachments/1196326223652077672/1199505979197362186/Screenshot_2024-01-23_at_7.00.46_PM.png?ex=65c2c9ff&is=65b054ff&hm=c719d42cad255ef88107930349d737dc85794d0c7a3819daa4779a6bd445a0ae&](url)

**Comment by naelstrof** [comment]
Yeah sorry! We don't have any Mac equipment to test, it appears to be a shader error. It might be a while for us to fix this one as the only way to test this is with a real Mac. It seems like it works on some macs and not others so it might take even longer! Thank you for supplying the mac model as that will help a lot.
